### ISSUE TITLE
Help With Configuration.yaml

### ISSUE THREAD

**Issue by scottkritz** [ajmarks/ajmarks_ha_components#3]
@ajmarks, thanks SO MUCH for building this! You are AWESOME.

So I know this is a very n00b question, but could you help me with what I need to add into `configuration.yaml` to get this working? 

I've already uploaded your `ge_kitchen` folder beneath `/config/custom_components`.

Apologies for asking such basic questions here.

Thanks in advance for all of your work on this + your help with these questions!

**Comment by ajmarks** [comment]
Just put  ge_kitchen under  /config/custom_components, and add the
integration through the control panel

**Comment by scottkritz** [comment]
@ajmarks, when you say "control panel" do you mean Configuration > Integrations? Or elsewhere? (If the former, I've restarted a few times now and am not seeing it there.)

Thanks again for the help.

**Comment by ajmarks** [comment]
Sorry, yes

**Comment by scottkritz** [comment]
Got it. Thought that’s what you meant, @ajmarks. 

Bummer, though, since I’m not seeing it there. 

I’m overdue an update to the core. So I’ll try that and see if it takes care of it. 

Just want to say thanks once more for building this and for the help here. REALLY appreciate it.

**Comment by scottkritz** [comment]
@ajmarks, the update to the HASS core did the trick! Very excited to have this integrated now. Gonna work on getting these ported over to Homekit via HA next. 

THANK YOU.

**Comment by scottkritz** [comment]
@ajmarks, would you ever turn the Github Sponsor feature on (or send me your Venmo/PayPal/etc.) so I can further support you with a small sum for doing this for the community? I know you built this for yourself, but sharing/supporting it for us all is a pretty special thing. And I'd like to support you further in a way that I can—if that is helpful/useful at all for you.

**Comment by ajmarks** [comment]
No, but thank you very much. All I'd ask is to please report any bugs you
find so we can make it better.

**Comment by scottkritz** [comment]
@ajmarks, well, that's the very least I can do. And I will certainly do that. You're AWESOME, my friend.

**Comment by zably** [comment]
Thanks so much for this!  I got the entities card working properly, but I was wondering how to get the "Oven Controls" similar to the one at the bottom of the readme (with the nice temperature vs. time graph)?    Is it dependent upon oven model?

Sorry if this doesn't belong here, I didn't want to make a new issue since I think this is just me being dumb.

**Comment by ajmarks** [comment]
I need to update that screenshot for some HA interface changes, but you
should be able to get the temperature graph on the history tab once you
click into the entity card

**Comment by zably** [comment]
Thanks!  Looks like it's probably my oven model then, I see something called "User Temp Offset" but no actual temperatures, so mine probably doesn't report that info.

**Comment by ajmarks** [comment]
Your oven probably supports the user temp offset, but it's not hugely
useful. That's something you can set in the oven's setting to manually
correct displayed temperatures, for example by having the oven always
display 5 degrees above what it's measuring (after the factory calibration)
